docs/_documentations/image-registry-credentials.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: Image registry guidance for Eclipse Codewind
+description: Image registry guidance for Eclipse Codewind
+keywords: users, projects, Kubernetes, image, registry, deployment, push
+duration: 5 minutes
+permalink: dockerregistry
+type: document
+parent: installoncloud
+order: 1
+---
+# Codewind Docker registry guidance
+When you run Codewind on Kubernetes, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and to push the images to an image registry that you specify. You can use the `Image Registry Manager` in your Codewind IDE to configure the registry to use. 
```

Comment:
Just spotted this, I think we need to clarify by changing `...on Kubernetes, Codewind uses..`  to something like `..on Kubernetes, for Codewind style projects, Codewind uses...`

This is because for non-codewind style projects, there is no pushing of images, and so no need to specify a push registry.

Reply:
Agreed with @jopit. This doc was originally written before Codewind on Kube supported Appsody and Odo projects.